libs/icu/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=icu4c
-PKG_VERSION:=63.1
+PKG_VERSION:=64.1
 PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-63_1-src.tgz
+PKG_SOURCE:=$(PKG_NAME)-64_1-src.tgz
 PKG_SOURCE_URL:=http://download.icu-project.org/files/$(PKG_NAME)/$(PKG_VERSION)
```

Comment:
Add GitHub as primary source per "suggestion" by upstream and keep old URL as fallback?
http://site.icu-project.org/download/64#TOC-ICU4C-Download (Links to GitHub)
https://github.com/unicode-org/icu/releases/tag/release-64-1

Reply:
I knew what you pointed out, I haven't made any changes since the past download sites were still valid.

The new japanese era name (reiwa) correspondence (64.2) should come out soon, so I change it at that time.

Reply:
fixed